pkg/util/paramtable/component_param.go

```diff
@@ -1782,8 +1782,8 @@ func (p *queryNodeConfig) init(base *BaseTable) {
 		Version:      "2.0.0",
 		DefaultValue: "1024",
```

Comment:
why not change the default value to 128?

Reply:
I'm not sure about possible side effects to modify this default value